package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.8.0",
   "dependencies": {
     "browserify": "5.9.1",
+    "browserify-optional": "^1.0.0",
```

Comment:
Please pick a specific version rather than a range here, to match the style used for other dependencies.